src/SourceBuild/content/Directory.Build.props

```diff
@@ -189,4 +189,9 @@
     <PoisonUsageReportFile>$(PackageReportDir)poison-usage.xml</PoisonUsageReportFile>
   </PropertyGroup>
 
-</Project>
+  <PropertyGroup>
+    <VerticalName Condition="'$(VerticalName)' == ''">DefaultVertical</VerticalName>
```

Comment:
I'm not sure if `vertical` is an appropriate term to use here. What this captures is a unique build identifier, right? cc @mmitche

Reply:
Would something like `ManifestName` make more sense? We have the same in Arcade's Publish.proj: https://github.com/dotnet/arcade/blob/87b015b938e5400d6e57afd7650348c17a764b73/src/Microsoft.DotNet.Arcade.Sdk/tools/Publish.proj#L70

Reply:
Most repos, like installer, that produce multiple manifests do not call each build a 'vertical', though in essence that is what they are. They also do often have a concept of a primary vertical, but there isn't really any formal term used. It's usually along the lines of: "/p:PublishNugetPackages=true". It's not consistent across repos.

You could just say "ManifestName" here. What differentiates this usage, I think, is that we're really formalizing the 'primary' vertical for join points, and we have to give that some kind of name. Primary/main vertical makes the most sense in that context, vs. "Primary Manifest" or such. Then I think it makes sense to use Vertical here too.